util/junitxml/junitxml_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024 Kushal Shukla
```

Comment:
```suggestion
// Copyright 2024 The Prometheus Authors
```
Note: using individual name was the reason we rejected the original PR.